File: Add Code Here/Dart/palindrome.dart
//Program that checks if a given number is palindrome or not

void main() {
  int n=1331;
  int copy = n;
  int rev=0;
  
  while(n!=0){
    rev=rev*10 + n%10;
    n=(n/10).round();
  }
  
  if(rev==copy)
  {
    print('Palindrome');
  }
  else
  {
    print('Not Palindrome');
  }
}

File: Add Code Here/C#/Palindrome.cs
using System;
					
public class Program
{
	public static void Main()
	{
		    string initial = "";
		    
		    Console.WriteLine("Enter a String to check for Palindrome");
 
            string input = Console.ReadLine();
 
            int Length = input.Length;
 
		    
            if (Length == 0)
            {
                Console.WriteLine("You did not enter the string");
 
            }
            else
            { 
                for (int j = Length - 1; j >= 0; j--)
                {
                    initial = initial + input[j];
                }				
 				
                if (initial == input)
                {
                    Console.WriteLine(input + " is palindrome Number");
                }
                else
                {
                    Console.WriteLine(input + " is not a palindrome Number");
                }
             
            }
	}
}
